### ISSUE TITLE
add the 'featured' cssClass to featured news

### ISSUE THREAD

**Issue by asaage** [contao/core#8232]
would be nice to have that in the default-Templates [like here](https://community.contao.org/de/showthread.php?17909-quot-Beitrag-hervorheben-quot-CSS-Klasse&p=257777&viewfull=1#post257777)

**Comment by Aybee** [comment]
:+1: 
Never used the featured option but seems to be a good idea to automatically have the class 'featured' within the news wrapper.
So
```html
<div class="layout_short arc_1 block first even">
```
becomes
```html
<div class="layout_short arc_1 block first even featured">
```

**Comment by fritzmg** [comment]
:+1:

**Comment by leofeyer** [comment]
Implemented in contao/news-bundle@a88b2abcf8a0dd0417e183d58284a12d0dc8b9f6.
